ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1301,9 +1301,7 @@ literals:
   - ON
   - OFF
 - defrost_temperature_sensor
-
-
-
+- room_temperature_sensor
```

Comment:
can the existing field `zone_air_temperature_sensor` be used instead?

Reply:
Yes, updated telemetry.